### PR TITLE
Adjustment UI update contact page  layout fix 

### PR DESCRIPTION
**Pull request by chandrahmani**
--- I have updated contact page layout padding and margin  then added some data 

--- I have added to component in footer terms & conditions page and privacy policy page

Screenshot 2023-03-13 233318

Screenshot 2023-03-13 233344

### CODE REVIEW

File: src/components/core/footer/Footer.jsx [hunk 1]
```diff
@@ -8,16 +8,23 @@ const Footer = () => {
...
-                </ul>
-              </div>
               <div className="number-wrap">
-                <a href="#" className="number">
-                  Mob: +917006036459 / +91990675515
-                </a>
+                <div className="policy">
+                  <ul>
+                    <li>
+                      <a href="http://127.0.0.1:5174/hello-travel/termcondition">
```
Comment (hidaytrahman) [inline src/components/core/footer/Footer.jsx:15]: Put router link instead of this

File: src/components/core/footer/Footer.jsx [hunk 2]
```diff
@@ -8,16 +8,23 @@ const Footer = () => {
...
+                        Terms & conditions
+                      </a>
+                    </li>
+                    <li>
+                      <a href="http://127.0.0.1:5174/hello-travel/privacypolicy"> Privacy policy</a>
```
Comment (hidaytrahman) [inline src/components/core/footer/Footer.jsx:20]: router link instead of anchor

File: src/components/core/footer/Footer.jsx [hunk 3]
```diff
@@ -8,16 +8,23 @@ const Footer = () => {
...
+                    </li>
+                  </ul>
+                </div>
+
+                <span className="number">
```
Comment (hidaytrahman) [inline src/components/core/footer/Footer.jsx:25]: classname will be phone number

File: src/components/views/contact/Contact.jsx [hunk 4]
```diff
@@ -1,59 +1,101 @@
...
+  WhatsappLogo
+} from 'phosphor-react';
 import React from 'react';
 import { Button, Input, Textarea } from 'react-carbonui';
 import './Contact.css';
 
 const Contact = () => {
   return (
     <div className="contact-wrapper">
+      <h2>Contact US</h2>
```
Comment (hidaytrahman) [inline src/components/views/contact/Contact.jsx:17]: Use h1 for all the main title of every page/view

**Review by hidaytrahman — APPROVED** [review]
do the adjustments

File: src/components/views/contact/Contact.jsx (hunk 4 above)
Comment (chandrahmani) [reply src/components/views/contact/Contact.jsx:17]: 👍

File: src/components/core/footer/Footer.jsx (hunk 3 above)
Comment (chandrahmani) [reply src/components/core/footer/Footer.jsx]: 👍

**Pull request merged**